### ISSUE TITLE
[Qualys] Can't find template ID

### ISSUE THREAD

**Issue by tom-dell** [HASecuritySolutions/VulnWhisperer#127]
Hi,

Running  `vuln_whisperer -c frameworks_example.ini -s qualys_web`

Results in 

```
ERROR:qualysWhisperAPI:Could not retrieve template ID
/usr/local/lib/python2.7/dist-packages/qualysapi/connector.py:162: FutureWarning: The behavior of this method will change in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
  if not data:
ERROR:main:Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.
ERROR: Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.
```

The template exists in Qualys. Below is my frameworks_example.ini file

------------------------------------------------------------------------------------

```
[qualys_web]
#Reference` https://www.qualys.com/docs/qualys-was-api-user-guide.pdf to find your API
enabled = true
hostname = [URL]
username = [username]
password = [password]
write_path=/opt/VulnWhisperer/data/qualys/
db_path=/opt/VulnWhisperer/data/database
verbose=true

# Set the maximum number of retries each connection should attempt.
# Note, this applies only to failed connections and timeouts, never to requests where the server returns a response.
max_retries = 10
# Template ID will need to be retrieved for each document. Please follow the reference guide above for instructions on how to get your template ID.
template_id = 1840

[qualys_vuln]
# Reference https://www.qualys.com/docs/qualys-was-api-user-guide.pdf to find your API
enabled = true
hostname = [URL]
username = [username]
password = [password]
write_path=/opt/VulnWhisperer/data/qualys/
db_path=/opt/VulnWhisperer/data/database
verbose=true

# Set the maximum number of retries each connection should attempt.
# Note, this applies only to failed connections and timeouts, never to requests where the server returns a response.
max_retries = 10
# Template ID will need to be retrieved for each document. Please follow the reference guide above for instructions on how to get your template ID.
# template_id = 126024
```

**Comment by reactor7171** [comment]
Hi @tom-dell,

We have had some issues lately due to a change that I did with qualysapi, failing to notice that it was breaking the qualys_web module of the program due to some dependencies to austin-taylor's qualysapi fork that are not on the original library, we are working on that.

Meanwhile if you want, you can try with an older version; 1.7.1 should work. Apologies for the inconvenience.

Cheers!

**Comment by reactor7171** [comment]
Hi @tom-dell, could you check the new PR and see if everything works as expected now? Otherwise we will consider it resolved with the #146 PR :)

Cheers!
